components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -2965,6 +2966,64 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+######################################################
+# The Tenant Theme resource APIs
+######################################################
+  /tenant-theme/import:
+
+    post:
+      security:
+        - OAuth2Security:
+            - apim:tenant_theme_manage
+      consumes:
+        - multipart/form-data
+      x-wso2-curl: "curl -k -F \"file=@exported.zip\" -X POST -H \"Authorization: Bearer
+      ae4eae22-3f65-387b-a171-d37eaa366fa8\" https://localhost:9443/api/am/admin/v1/tenant-theme/import?tenantDomain=abc
+      .com"
+      x-wso2-request: |
+        POST https://localhost:9443/api/am/admin/v1/tenant-theme/import
+        Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+      x-wso2-response: "HTTP/1.1 200 OK\nTenant Theme imported successfully."
+      summary: Import a DevPortal Tenant Theme
+      description: |
+        This operation can be used to import a DevPortal tenant theme.
+      parameters:
+        - name: file
+          in: formData
+          description: |
+            Zip archive consisting of tenant theme configuration
+          required: true
+          type: file
+        - name: tenantDomain
+          in: query
```

Comment:
Do we really have to provide the tenant domain name? Shouldn't we extract the tenant domain from the auth context? Unless an unauthorized tenant user can update the tenant theme of some other tenant domain.

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/8595/commits/6ae0bc73c2aefce667fc3908529e7750aaa25d97